contracts/0.8.9/oracle/AccountingOracle.sol

```diff
@@ -232,6 +233,10 @@ contract AccountingOracle is BaseOracle {
         /// at the reference slot.
         uint256 elRewardsVaultBalance;
 
+        /// @dev The shares amount requested to burn through Burner as observed
+        /// at the reference slot.
```

Comment:
maybe tell here how to obtain it? it's very impl-specific value anyway so referring the specific fn won't make the docs more fragile imo

Reply:
Yeah, added some more explanations :+1: